### PR TITLE
Press kit pages

### PR DESCRIPTION
**Pull request by NistorCristian**
I created 3 pages:

Press kit, Press kit for Status and Brand materials for Status.
Changed some texts based on @j-zerah request.

### CODE REVIEW

File: themes/navy/layout/press-kit-status-press.ejs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
....41799 4.95237 5.16402 5.20634C4.99471 5.37566 4.78307 5.46031 4.5291 5.46031ZM17.1429 3.64021C17.0159 3.64021 16.8465 3.59788 16.7195 3.51322C16.3809 3.34391 16.0847 3.1746 15.746 3.04761C15.2804 2.8783 15.0688 2.37037 15.2804 1.90476C15.4497 1.43915 15.9577 1.22751 16.4233 1.43915C16.8043 1.60846 17.2275 1.77778 17.5661 1.98941C17.9894 2.20106 18.1587 2.75132 17.9048 3.1746C17.7778 3.4709 17.4815 3.64021 17.1429 3.64021ZM7.99999 3.13227C7.66138 3.13227 7.32277 2.92063 7.19578 2.58201C7.02646 2.1164 7.23806 1.60846 7.66138 1.43915C8.04235 1.26984 8.46559 1.14286 8.88891 1.01587C9.35451 0.888886 9.82011 1.14286 9.98938 1.60846C10.1587 2.07407 9.86246 2.53968 9.39679 2.70899C9.05818 2.79365 8.71957 2.92063 8.38096 3.08995C8.21162 3.13227 8.08463 3.13227 7.99999 3.13227ZM13.2064 2.41269C13.164 2.41269 13.164 2.41269 13.1217 2.41269C12.7831 2.37037 12.4021 2.37037 12.0635 2.37037C11.5979 2.37037 11.1746 1.98941 11.1746 1.48148C11.1746 1.01587 11.5555 0.59259 12.0212 0.59259C12.0212 0.59259 12.0635 0.59259 12.1058 0.59259C12.4868 0.59259 12.91 0.634918 13.3334 0.677246C13.799 0.719573 14.1799 1.14286 14.0952 1.65079C14.0529 2.07407 13.672 2.41269 13.2064 2.41269Z" fill="white"/> </svg>
```
Comment (jakubgs) [inline themes/navy/layout/press-kit-status-press.ejs]: Can we extract the SVGs into files? This doesn't make for very readable HTML template.

**Review by jakubgs — CHANGES_REQUESTED** [review]
Looks fine, but don't commit `package-lock.json`.

File: themes/navy/layout/press-kit-status-press.ejs (hunk 1 above)
Comment (NistorCristian) [reply themes/navy/layout/press-kit-status-press.ejs]: I've replaced 10 svgs with images and also removed package-lock.json

**Review by jakubgs — APPROVED** [review]
Awesome.

**Comment by jakubgs** [comment]
Oh, your last commit isn't signed. Mind squashing this into one signed commit? I'll merge then.

**Comment by jakubgs** [comment]
Whatever, I squashed it myself.

**Comment by jakubgs** [comment]
Deployed in https://ci.status.im/job/misc/job/dev.statusnetwork.com/19/.

Looking good: https://dev.statusnetwork.com/press-kit/

**Pull request merged**